### DISCUSSION TITLE
Problema con los scripts importados nextjs

### DISCUSSION THREAD

**Discussion by alainmy** [vercel/next.js#14459] [Help]
en mi _document.js tengo importado unos scripts que usa la plantilla con la que trabajo
cuando la pagina principal carga estos se cargan y funciona todo. Pero cuando navego hacia orta pagina usando Link de nextjs como este carga lo nesesario no me vuelve a carga los scripts, por lo que pierde funcionalidades los componentes q dependen de ellos

import Document, { Html, Head, Main, NextScript } from 'next/document'

export default class MyDocument extends Document {
  render() {
    return (
      <Html lang="en">
        <Head />
        <body>
          <Main />
          <script src="assets/jquery-3.5.1.min.js" type="text/javascript"></script>
          <script src="assets/vendors/base/vendors.bundle.js" type="text/javascript"></script>
          <script src="assets/demo/default/base/scripts.bundle.js" type="text/javascript"></script>
          <NextScript />
        </body>
      </Html>
    )
  }
}

**Comment by dsauna** [comment]
Estas seguro que el _document.js fue creado en el mismo folder que pages?
